.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gcr.io/oss-fuzz-base/base-builder:v1
+COPY . $SRC/app-boilerplate
+COPY ./.clusterfuzzlite/build.sh $SRC/
+RUN git clone https://github.com/LedgerHQ/ledger-secure-sdk.git $SRC/app-boilerplate/BOLOS_SDK
```

Comment:
Unfortunately, this has to be used.

Reply:
I think it could be better to use a "fuzzing" docker based on our [app-builder](https://github.com/LedgerHQ/ledger-app-builder) which is updated with the last SDK used. The fuzzing docker will just include the additional tools/packages for fuzzing.

In addition you can remove this repository
https://github.com/LedgerHQ/app-boilerplate-fuzz-storage

Reply:
Why do you declare a docker here? It seems not called?

Reply:
I've updated the Dockerfile to use the latest SDK